src/Tizen.WebView/Tizen.WebView/WebView.cs

```diff
@@ -328,6 +425,180 @@ public void SetFocus(bool focused)
             Interop.ChromiumEwk.ewk_view_focus_set(_realHandle, focused);
         }
 
+        /// <summary>
+        /// Gets size of the content.
+        /// </summary>
+        /// <returns> size of the coordinate.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public Size ContentsSize
+        {
+            get
+            {
+                int width, height;
+                Interop.ChromiumEwk.ewk_view_contents_size_get(_realHandle, out width, out height);
+                return new Size(width, height);
+            }
+        }
+
+        /// <summary>
+        /// Get cookies for URL.
+        /// </summary>
+        /// <param name="url">The URL.</param>
+        /// <returns>'cookie string' on success, otherwise 'null'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string GetCookies (string url)
+        {
+            return Interop.ChromiumEwk.ewk_view_get_cookies_for_url(_realHandle, url);
+        }
+
+        /// <summary>
+        /// Exit full screen.
+        /// </summary>
+        /// <returns>'true' on success, otherwise 'false'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool ExitFullscreen ()
```

Comment:
It should be void, in C# we don't return boolean on success.

Reply:
Done